tests/testapp/blocks.py

```diff
@@ -309,3 +326,4 @@ class StreamFieldBlock(blocks.StreamBlock):
     block_with_name = BlockWithName()
     advert = SnippetChooserBlock("testapp.Advert")
     person = SnippetChooserBlock("testapp.Person")
+    block_with_custom_interface = BlockWithCustomInterface()
```

Comment:
nitpick: the convention is to end with what it is. so something like `CustomInterfaceBlock`. 

and `CustomInterfaceForBlock` -> `CustomBlockInterface`. I had a few moment expanding the GraphQL query locally, and exploring the schema in GraphiQL